GaeManda/Projects/Presentation/OnBoarding/Implementations/DogSetting/SecondDogSetting/SecondDogSettingViewController.swift

```diff
@@ -149,6 +171,20 @@ final class SecondDogSettingViewController:
 			.disposed(by: disposeBag)
 	}
 	
+	private func bindDropDown() {
+		dogBreedDropDownView.selectedOptionRelay
+			.bind(with: self) { owner, selectedOption in
+				owner.dogBreedDropDownButton.setTitle(selectedOption, for: .option)
+			}
+			.disposed(by: disposeBag)
+		
+		dogCharacterDropDownView.selectedOptionRelay
+			.bind(with: self) { owner, selectedOption in
+				owner.dogCharacterDropDownButton.setTitle(selectedOption, for: .option)
+			}
+			.disposed(by: disposeBag)
```

Comment:
내눈좋

더 개선한다면 요정도~?
```swift
// DropDownView.swift
public extension Reactive where Base: DropDownView {
	var selectedOption: ControlEvent<String> {
		let source = base.dropDownTableView.rx.itemSelected.map { base.dataSource[$0.row] }
		return ControlEvent(events: source)
	}
}

// DropDownButton.swift
public extension Reactive where Base: DropDownButton {
	var title: Binder<(title: String?, mode: Base.DropDownTextMode)> {
		return Binder(self.base) { button, arg in
			button.setTitle(arg.title, for: arg.mode)
		}
	}
}

```

```suggestion
	dogCharacterDropDownView.rx.selectedOption
			.map { ($0, .option) }
			.bind(to: dogCharacterDropDownButton.rx.title)
			.disposed(by: disposeBag)	
```